--- controllers.h~	2019-03-13 13:57:55.526199900 +1100
+++ controllers.h	2019-03-13 13:58:54.917954084 +1100
@@ -18,7 +18,6 @@
 #define __CONTROLLERS_H
 
 #include "synth.h"
-#include "../Dexed.h"
 #include <stdio.h>
 #include <string.h>
 
@@ -116,7 +115,6 @@
         if ( ! ((wheel.eg || foot.eg) || (breath.eg || at.eg)) )
             eg_mod = 127;
         
-        TRACE("controllers refresh>>> amp_mod %d pitch_mod %d", amp_mod, pitch_mod);
     }
     
     FmCore *core;
--- env.cc~	2019-03-13 13:57:55.526199900 +1100
+++ env.cc	2019-03-13 13:59:09.749902426 +1100
@@ -20,7 +20,6 @@
 #include "synth.h"
 #include "env.h"
 
-#include "../Dexed.h"
 //using namespace std;
 
 uint32_t Env::sr_multiplier = (1<<24);
